deployment/app/src/main/java/com/rk/amii/activities/AuthenticationActivity.java

```diff
@@ -60,6 +61,25 @@ protected void onCreate(Bundle savedInstanceState) {
                             (Boolean) loginResult.getOrDefault("is_agreed_to_privacy_policy", null)
                     );
                     startActivityForResult(intent, 100);
+
+                    JSONObject response = service.getUserProfile();
```

Comment:
are these changes related to current ticket?

Reply:
@dimasciput It is related since the upload preference is located User Profile. So they use same endpoint, which is endpoint to update/get user profile.